static/src/javascripts/projects/journalism/views/audioSeriesContributions.html

```diff
@@ -8,7 +8,7 @@ <h2>Support The Guardian</h2>
                     <br> journalism the world needs</p>
                 <a class="contributions__contribute contributions__option-button"
                    data-link-name="audio-series-contributions"
-                   href="<%= supportContributeURL %>">
+                   href="<%= supportContributeURL %>?acquisitionData=%7B%22componentType%22%3A%22ACQUISITIONS_OTHER%22%2C%22source%22%3A%22GUARDIAN_WEB%22%2C%22campaignCode%22%3A%22today_in_focus%22%2C%22componentId%22%3A%22audio_series_page%22%7D&INTCMP=today_in_focus">
```

Comment:
Do the `componentType`, `campaignCode` and `INTCMP` have to be agreed beforehand? Maybe they already have, in which case ignore this!

Reply:
Yes, they've been given to us by the RR team 🤗